.github/workflows/pr-actions.yml

```
@@ -58,6 +58,9 @@ jobs:
     name: "Test: End to End"
     runs-on: ubuntu-latest
     needs: test-unit
+    strategy:
```

```suggestion
    strategy:
      # when one test fails, DO NOT cancel the other
      # containers, because this will kill Cypress processes
      # leaving the Dashboard hanging ...
      # https://github.com/cypress-io/github-action/issues/48
      fail-fast: false
```
Looks like you also need this just in case, https://github.com/cypress-io/github-action#parallel.